docs/examples/cmark_spec_0.30.yml

```diff
@@ -3494,13 +3627,14 @@ cases:
         - type: html
           value: '  <div>'
         - type: code
-          lang: ''
+          lang: null
+          meta: null
           value: <div>
     myst: |2
         <div>
 
           <div>
-    html: |2-
+    html: |-2
```

Comment:
This change is not correct, due to the regex replacement I used, as a temporary solution, but not sure how to output this format using js-yaml

Reply:
I can remove the regex replacement, but then obviously there will be more "spurious" diffs of `|` vs `|-`